client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -97,6 +98,7 @@ export const siteSetupFlow: Flow = {
 		const isEnabledFTM = isEnabled( 'signup/ftm-flow-non-en' ) || isEnglishLocale;
 		const urlQueryParams = useQuery();
 		const isPluginBundleEligible = useIsPluginBundleEligible();
+		const isMobile = useViewportMatch( 'medium', '<' );
```

Comment:
I'm wondering why we use `< medium` here because in other areas in Calypso we use:

- `const isMobile = useViewportMatch( 'small', '<' );`
- `const isMobile = ! useViewportMatch( 'small' );`

Also we have `isMobile()` from `@automattic/viewport`.

Reply:
I make breakpoint at 782px (medium) to match `.design-picker__grid` [here](https://github.com/Automattic/wp-calypso/blob/87453f2b29ada97fac1766c23a23477a59aae5ed/packages/design-picker/src/components/style.scss#L405)

Reply:
I think it's odd to name it as `isMobile` since in the context of Stepper, mobile usually refers to < 600px. That said, it depends on if you want to make the PA available for 600px - 782px resolution.

Reply:
So should the variable name be updated in this case?

Reply:
I recommend using `const isDesktop = useViewportMatch( 'large' );` 🙂

Reply:
To sum up:
> 960px - CTA takes users to PA.
≤ 960px - CTA takes users to the site editor with BC.